src/java/io/bazel/rulesscala/io_utils/DeleteRecursively.java

```diff
@@ -1,26 +0,0 @@
-package io.bazel.rulesscala.io_utils;
-
-import java.io.IOException;
-import java.nio.file.*;
-import java.nio.file.attribute.BasicFileAttributes;
-
-public class DeleteRecursively {
```

Comment:
why not keep this implementation? Sounds better without the sort

Reply:
> why not keep this implementation? Sounds better without the sort

I left the stream based version, because I felt it has less boilerplate code

Reply:
On the other hand it sorts and this one doesn't so it sounds more performant and we're already trying to optimize our perf